Azure-Samples/Azure-Time-Series-Insights
# use netstandard2.0

## Purpose

This PR moves the Data Plane client project to NetStandard2.0

## Does this introduce a breaking change?
```
[ ] Yes
[x] Maybe
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ x ] Code style update (formatting, local variables)
[ x ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
* Check this works on your builds. There's no CI attached to this repo (afaik) so it's hard to tell whether this is a breaking change or not.

## What to Check

* From the big csproj files, I assume you're using Visual Studio on Windows. Can you check you can build?
* Are the csproj files auto-generated, and will they be overwritten in the future?
